src/groups/mqb/mqbblp/mqbblp_routers.h

```diff
@@ -807,12 +815,14 @@ class Routers {
 inline Routers::Consumer::Consumer(
     const bmqp_ctrlmsg::StreamParameters& streamParameters,
     unsigned int                          subQueueId,
+    bool                                  isBroadcastBroker,
```

Comment:
The name `isBroadcastBroker` doesn't match the arg's name in declaration (line 383): `isFirstHop`